lib/Gitlab/Api/Projects.php

```diff
@@ -736,6 +736,19 @@ public function removeVariable($project_id, $key)
         return $this->delete($this->getProjectPath($project_id, 'variables/'.$this->encodePath($key)));
     }
 
+    /**
+     * @param int $project_id
```

Comment:
why is the $project_id always typed as integer even it's allowed to use project_path as well?

Reply:
This PR is almost three years old, I have no idea!